frontends/mit-open/src/pages/ListDetailsPage/ListDetailsPage.tsx

```diff
@@ -108,8 +18,8 @@ const ListDetailsPage: React.FC<ListDetailsPageProps> = ({
       className="learningpaths-page"
     >
       <MetaTags title={list?.title} />
-      <Container maxWidth="sm">
-        <ListDetailsComponent
+      <Container maxWidth="md">
+        <ItemsListingComponent
```

Comment:
Not a blocker, but I think we could use some padding on the top of this page above the header. The padding used to exist on the component itself, but was taken out so that it displays properly within the dashboard. I think this page would look a lot better if we just added some spacing there. There's no design for this page, so just use your best judgement.

![image](https://github.com/user-attachments/assets/0bd6c411-a54e-4b57-945a-ca3a392f696a)

Reply:
:+1: Added some margin